Kontent.Ai.Management.Tests/ManagementClientTests/PublishingTests.cs

```diff
@@ -0,0 +1,335 @@
+﻿using FluentAssertions;
+using Kontent.Ai.Management.Models.LanguageVariants;
+using Kontent.Ai.Management.Models.Publishing;
+using Kontent.Ai.Management.Models.Shared;
+using Kontent.Ai.Management.Models.Workflow;
+using Kontent.Ai.Management.Tests.Base;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Net.Http;
+using Xunit;
+using static Kontent.Ai.Management.Tests.Base.Scenario;
+
+namespace Kontent.Ai.Management.Tests.ManagementClientTests;
+
+public class PublishingTests
+{
+    private readonly Scenario _scenario;
+
+    public PublishingTests()
+    {
+        _scenario = new Scenario(folder: "Workflow");
```

Comment:
Just a question. Is it needed to define "Workflow" folder if no test uses .json file from `Data` folder here?

Reply:
I think it's not necessary at all. On the other hand, a scenario is needed and it takes the folder as a parameter. I do not want to modify the scenario because of this one exception.